### DISCUSSION TITLE
Importer skips duplicate transactions that do no exist (anymore)

### DISCUSSION THREAD

**Discussion by chevdor** [firefly-iii/firefly-iii#9386] [Firefly III] [answered]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [x] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem
- [x] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I did run an faulty import with mapping issues.
I then deleted the transactions using the import tag (very handy in that case btw !).
While redoing a better import, a few tx are skipped with an error:
```
Line #0: transactions.0.description: Duplicate of transaction #5734. (original value: "(empty description)")
...
```

However, checking my installation at: https://firefly.somedomain.org/transactions/show/5734 results in no transaction being found.

### Debug information

I am using `fireflyiii/data-importer:version-1.5` which currently resolves to 1.5.6.
FF is v6.1.21

### Expected behaviour

Transactions should not be skipped due to a duplicate that does not exist.

**Comment by JC5** [comment, marked as answer]
Hi, thanks for starting a discussion.

Please [refer to the documentation](https://docs.firefly-iii.org/how-to/data-importer/import/duplicates/#deleted-but-duplicate) for an explanation about this.

**Comment by chevdor** [comment]
Oh wow, I did not expect that one, especially after reading:
<img width="409" alt="image" src="https://github.com/user-attachments/assets/b7cc7b3b-549d-48c6-8972-cc6f463f6217">

Thanks for the super fast response, impressive as usual !

**Reply by JC5** [reply]
Yeah that's confusing, but on the other hand I haven't got a undo button yet so for all intents and purposes it's true 😉

**Comment by chevdor** [comment]
Would it be possble to show the tx as deleted when visiting  https://firefly.somedomain.org/transactions/show/5734  ? That would also have put me on the track that it is not really deleted (yet). It could even be a generic page without the details and pointing to the /purge function.

A quick solution could also be to:
- in case the importer runs into error, show a "check out those typical issue" link pointing to a discussion or to the doc, or even to a list of issues based on a tag
- listing those typical cases

I would probably have found the topic then, purge, and try again the import, and not open an issue :)
